showcases/dictionaries-and-secret-stores/terraform/main.tf

```diff
@@ -0,0 +1,162 @@
+# Configure the Microsoft Azure Provider
+provider "azurerm" {
+  subscription_id = "${var.subscription_id}"
+  client_id       = "${var.client_id}"
+  client_secret   = "${var.client_secret}"
+  tenant_id       = "${var.tenant_id}"
+}
+
+# Create a resource group if it doesn’t exist
+resource "azurerm_resource_group" "myterraformgroup" {
```

Comment:
can we have a betternames instaed of `my*` names, else just remove the `my` prefix from names

Reply:
'my' removed